saltswap/tests/test_simulation.py

```diff
@@ -1,11 +1,191 @@
 from simtk import openmm, unit
-from openmmtools.testsystems import WaterBox
+from openmmtools.testsystems import WaterBox, DHFRExplicit
 from simtk.openmm import app
 import numpy as np
-from saltswap import swapper, mcmc_samplers
+from saltswap import swapper, wrappers
 from openmmtools import integrators as open_integrators
 from saltswap.integrators import GHMCIntegrator
 
+
+
+class TestSalinator(object):
+    """
+    Tests the functionality of wrapper for the MCMC addition and deletion of salt.
+    """
+
+    def _create_protein_system(self, npert=10):
+        """
+        Creates a DHFR system to re-use in tests.
+        """
+        # System parameters
+        temperature = 300. * unit.kelvin
+        timestep = 2. * unit.femtoseconds
+        collision_rate = 90. / unit.picoseconds
```

Comment:
Because this is only used for unit tests, where I don't minimize, I want the simulations to be extra stable. As that's not's not clear, I'll add that as a comment.